frontend/src/routes/_layout/submissions/_layout.tsx

```diff
@@ -102,9 +96,8 @@ function SubmissionsLayoutRoute() {
           </div>
         </div>
       </div>
-
       {/* Desktop Layout */}
-      <div className="hidden md:grid md:grid-cols-4 gap-4 lg:gap-8 overflow-hidden px-4 lg:px-8">
+      <div className="hidden md:grid md:grid-cols-4 md:max-w-[1440px] mx-auto gap-4 lg:gap-8 overflow-hidden px-4 lg:px-8">
```

Comment:
Same as above.

If we can make a common "Container" component that keeps the width consistent and goes under Hero, we should